src/main/webapp/dev/js/common/updateCharCount.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Updates the number of characters left in the text area
+ * @param textArea - Text area for which char are to be counted
+ * @param letterCountId - Id of Label to display length of text area
+ */
+function updateCharLeftCount(textArea, letterCountId) {
+    const letterCountArea = $(`#${letterCountId}`);
+    const maxLength = textArea.attr('maxLength');
+    const charLength = textArea.val().length;
+    letterCountArea.text(maxLength - charLength);
+}
+/**
+ * Create the div to display the number of characters left in textArea
+ * @param textArea - Text area for which char are to be counted
+ * @param letterCountId - Id of area for letter count display
+ */
+function insertLetterCountArea(textArea, letterCountId) {
+    const maxLength = textArea.attr('maxlength');
```

Comment:
Hmm why is it not `maxLength` here as well?

Reply:
this is because the attribute name is 'maxlength' instead of 'maxLength'.